api/routes.go

```diff
@@ -62,7 +62,6 @@ func Routes(s *bittorrent.Service) *gin.Engine {
 	r.GET("/status", Status)
 
 	r.Any("/info", s.ClientInfo)
-	r.Any("/info/*ident", s.ClientInfo)
```

Comment:
Removing this will make you unable to call "/info/".

Reply:
it is not like we use it, but ok, i will revert.